remoteworkitem/trackerquery_repository_test.go

```diff
@@ -81,20 +91,26 @@ func (test *TestTrackerQueryRepository) TestExistsTrackerQuery() {
 		err := test.trackerRepo.Create(ctx, &tracker)
 		require.NoError(t, err)
 
-		query, err := test.queryRepo.Create(ctx, "abc", "xyz", tracker.ID, space.SystemSpace)
-		require.NoError(t, err)
+		query := remoteworkitem.TrackerQuery{
+			Query:     "abc",
+			Schedule:  "xyz",
+			TrackerID: tracker.ID,
+			SpaceID:   testFxt.Spaces[0].ID,
+		}
+		err = test.queryRepo.Create(ctx, &query)
+		assert.Nil(t, err)
 
 		err = test.queryRepo.CheckExists(ctx, query.ID)
-		require.NoError(t, err)
+		assert.Nil(t, err)
 	})
 
 	t.Run("tracker query doesn't exist", func(t *testing.T) {
 		req := &http.Request{Host: "localhost"}
 		params := url.Values{}
 		ctx := goa.NewContext(context.Background(), nil, req, params)
 
-		err := test.queryRepo.CheckExists(ctx, "11111111111")
-		require.IsType(t, errs.NotFoundError{}, err)
+		err := test.queryRepo.CheckExists(ctx, uuid.NewV4())
+		require.IsType(t, errors.NotFoundError{}, err)
```

Comment:
Please check for `require.Error(t, err)` first before checking for the type.

Reply:
done here 2eed5aa